queries/m68k/highlights.scm

```diff
@@ -0,0 +1,73 @@
+(symbol) @variable
+
+(label name: (symbol) @constant)
+
+[
+  (instruction_mnemonic)
+  (directive_mnemonic)
+] @function.builtin
+
+(include (directive_mnemonic) @include)
+(include_bin (directive_mnemonic) @include)
+(include_dir (directive_mnemonic) @include)
+
+
+(size) @attribute
+
+(macro_definition name: (symbol) @function.macro)
+(macro_call name: (symbol) @function.macro)
+
+[
+  (path)
+  (string_literal)
+] @string
+
+[
+  (decimal_literal)
+  (hexadecimal_literal)
+  (octal_literal)
+  (binary_literal)
+] @number
+
+[
+  (reptn)
+  (carg)
+  (narg) 
+  (macro_arg)
+] @variable.builtin
+
+[
+  (control_mnemonic)
+  (address_register)
+  (data_register)
+  (float_register)
+  (named_register)
+] @keyword
+
+(repeat (control_mnemonic) @repeat)
+(conditional (control_mnemonic) @conditional)
+
+(interpolated (macro_arg) @embedded) 
```

Comment:
there is no `@embedded`. You probably want to use `@none`

Reply:
I've removed it. The previous query mapping `macro_arg` to `variable.builtin` is fine in this context too.

Reply:
You probably meant to use `injections.scm`. See how injections work for C. It is not in `highlight.scm` for our repo

Reply:
Thanks for that. I had a look and I don't think it's really suitable here as it's a different use case to C's macro args. I'm happy to explain in more detail, but in short I think it makes sense to leave all macro args highlighted as variables. I did however add injections for comments and I've now added locals queries too, which I've tried out with `nvim-treesitter-refactor`.